parachain/pallets/control/src/lib.rs

```diff
@@ -305,7 +307,8 @@ pub mod pallet {
 		fn submit_outbound(message: Message) -> DispatchResult {
 			let ticket =
 				T::OutboundQueue::validate(&message).map_err(|_| Error::<T>::SubmissionFailed)?;
-			T::OutboundQueue::submit(ticket).map_err(|_| Error::<T>::SubmissionFailed)?;
+			T::OutboundQueue::submit(ticket, Priority::High)
```

Comment:
I think everything except the `CreateAgent` command should be high priority. So perhaps make `submit_outbound` accept a priority parameter.

Reply:
I would assume there could still be ACL control for `CreateAgent` and it's different from the normal operation from any signed origin(e.g. sending token from source chain).   

So I would prefer all commands in the control pallet with high priority(btw: maybe upgrade/pause should be with more priority like `emergency` though don't support yet).